docs/source/utils.rst

```diff
@@ -113,10 +113,78 @@ This detects ``address`` format and selects the socket family automatically.
     sock = bind_socket(address="::1", port=1234)
 
 
+Recurring callback
+++++++++++++++++++
+
+Runs coroutine function periodically with used defined strategy.
```

Comment:
Perhaps you meant "Runs coroutine function periodically with user-defined strategy."